package.json

```diff
@@ -176,7 +176,7 @@
     "overrides": {
         "minimist@1.2.5": "1.2.6",
         "follow-redirects": "1.15.6",
-        "es5-ext": "0.10.53"
+        "es5-ext": "0.10.63"
```

Comment:
add this package to your PR name please so we know it got bumped

Reply:
yep, updated that in PR name